api/v1/articles/index.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @defgroup api_v1_articles Article API requests
+ */
+
+/**
+ * @file api/v1/articles/index.php
```

Comment:
In the case of component handlers, it's derived directly from the handler name in the URL. See the docs at the head of `PKPComponentRouter`. (Not proposing we borrow this directly, just that I dislike the `index.php` wrappers in general -- we already have whitelisting elsewhere, so they're redundant.)